packages/perseus/src/styles/widgets/orderer.css

```diff
@@ -33,18 +45,28 @@
 .orderer.layout-horizontal .bank {
     padding: 0;
     margin: 0px 13px;
+    /* Fix for wrapping issue caused by CSS reset removal */
+    box-sizing: border-box;
+    width: 100%;
+    display: flex;
+    flex-wrap: nowrap;
+    overflow-x: auto;
+    white-space: nowrap;
 }
 .orderer div.paragraph {
     margin: 0;
 }
 .orderer .card-wrap {
     width: auto;
+    /* Fix for wrapping issue caused by CSS reset removal */
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
 }
 .orderer.layout-horizontal .card-wrap {
-    float: left;
-}
-.orderer.layout-horizontal .card-wrap:not(:first-child) {
-    margin-left: 8px;
+    flex-shrink: 0;
+    display: inline-block;
+    margin: 4px; /* Uniform spacing around each card */
```

Comment:
This kind of seems like a regression. Originally the first element wouldn't have left-side padding and now it will have 4px right?